src/exchange/wizard-hosted-creation/add-domain-name/add-domain-name.html

```diff
@@ -0,0 +1,231 @@
+                                data-ng-change="$ctrl.homepage.mxRelay = 'mx1.mail.ovh.net'"
+                                data-ng-value="true">
+                        <span data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_ovhDomain_isExchangeOnly')"></span>
+                    </label>
+                </div>
+                <div class="radio">
+                    <label>
+                        <input type="radio"
+                                data-ng-disabled="$ctrl.shouldDisabledDomainSelection"
+                                data-ng-model="$ctrl.homepage.domainIsAssociatedToEmailService"
+                                data-ng-value="false"
+                                data-ng-change="$ctrl.homepage.mxRelay = null">
+                        <span data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_ovhDomain_isNotExchangeOnly')"></span>
+                    </label>
+                </div>
+
+                <p role="alert"
+                   class="alert alert-info"
+                   data-ng-if="$ctrl.homepage.domainIsAssociatedToEmailService === true"
+                   data-ng-bind-html="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_ovhDomain_isEmailService')">
+                </p>
+
+                <form name="$ctrl.mxRelayForm"
+                        data-ng-class="{'has-error': $ctrl.mxRelayForm.$dirty && $ctrl.mxRelayForm.$invalid}"
+                        data-ng-if="$ctrl.homepage.domainIsAssociatedToEmailService === false">
+                    <p class="alert alert-info mb-2" role="alert" data-ng-bind-html="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_ovhDomain_nonAuthoritative')"></p>
+                    <label class="mt-3"
+                            for="mxRelay"
+                            class="control-label"
+                            data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_ovhDomain_mxRelay_header')">
+                    </label>
+                    <div class="form-group">
+                        <div class="w-25">
+                            <input type="text"
+                                   id="mxRelay"
+                                   name="mxRelay"
+                                   class="form-control"
+                                   required
+                                   data-ng-model="$ctrl.homepage.mxRelay"
+                                   data-ng-pattern="/^(?:[a-zA-Z0-9]+(?:[-_][a-zA-Z0-9]+)*)(?:(?:\.|\+)(?:[a-zA-Z0-9]+(?:[-_][a-zA-Z0-9]+)*))*(?:\.[a-zA-Z]{2,})$/"
+                                   data-ng-disabled="$ctrl.shouldDisabledDomainSelection">
+                            <p data-ng-if="$ctrl.mxRelayForm.mxRelay.$dirty && $ctrl.mxRelayForm.mxRelay.$invalid && $ctrl.mxRelayForm.mxRelay.$error.pattern"
```

Comment:
[ngMessages](https://docs.angularjs.org/api/ngMessages/directive/ngMessages) could make it more readable.